aiobotocore/httpsession.py

```diff
@@ -195,20 +215,15 @@ async def send(self, request):
             # https://github.com/boto/botocore/issues/1255
             headers_['Accept-Encoding'] = 'identity'
 
-            chunked = None
-            if headers_.get('Transfer-Encoding', '').lower() == 'chunked':
-                # aiohttp wants chunking as a param, and not a header
-                headers_.pop('Transfer-Encoding', '')
-                chunked = True
-
             if isinstance(data, io.IOBase):
                 data = _IOBaseWrapper(data)
 
             url = URL(url, encoded=True)
-            response = await self._session.request(
+            session = await self._get_session(proxy_url)
+            response = await session.request(
                 request.method,
                 url=url,
-                chunked=chunked,
+                chunked=self._chunked(headers_),
```

Comment:
I'm not sure we need to deal with the chunked parameter at all. To me it seems that [aithttp is able to handle that internally](https://github.com/aio-libs/aiohttp/blob/0e91eb0bd94298e501084f245df44cc1772c49e5/aiohttp/client_reqrep.py#L435) at least since https://github.com/aio-libs/aiohttp/commit/2c858346b6b82a703e9b187ee26a95f3f6772a67.

Reply:
hmm, seems like there's test coverage so it seems like you could pull it out to see what happens (assuming it's an integration test to ensure it's testing w/ aiohttp and moto)

Reply:
although I'd open an issue with comments for better testing in this area as well to do later unless you feel very motivated :)

Reply:
> although I'd open an issue with comments for better testing in this area as well to do later unless you feel very motivated :)

Unit tests in what's still an unfamiliar codebase are not my strong suit :-/

Reply:
> Let's see what happens: #1096

CI doesn't complain, but I'm not sure that's enough evidence. Maybe aiohttp folks can weigh in?

Reply:
we can just add a test later, i think werkzerg via moto should be able to handle it